example/x11.c

```diff
@@ -506,6 +506,19 @@ int main(int argc, char *argv[])
         libssh2_channel_free(channel);
         channel = NULL;
     }
+
+    if(session) {
+        libssh2_session_disconnect(session, "Normal Shutdown");
+        libssh2_session_free(session);
+    }
```

Comment:
```suggestion
    if(session)
        session_shutdown(session);
```
Use the local function doing the above two calls?